### ISSUE TITLE
Broken clinical signs widget for Down's Syndrome

### ISSUE THREAD

**Issue by ireneisdoomed** [opentargets/issues#2923]
**Describe the bug**
The Clinical Signs and Symptoms widget for [Down's Syndrome](https://platform.opentargets.org/disease/EFO_0001064/) is broken, showing an error.

@chinmehta has identified that there are records in the Clinical Signs dataset whose phenotype identifier is null, which should not be the case.

**Observed behaviour**

<details>
<summary>API response:</summary>

```json
{
  "data": {
    "disease": {
      "id": "EFO_0001064",
      "phenotypes": {
        "count": 51,
        "rows": [
          {
            "phenotypeHPO": {
              "id": "HP_0000474"
            },
            "phenotypeEFO": {
              "id": "HP_0000474"
            }
          },
          {
            "phenotypeHPO": {
              "id": "HP_0001674"
            },
            "phenotypeEFO": null
          },
          {
            "phenotypeHPO": null,
            "phenotypeEFO": null
          },
          {
            "phenotypeHPO": {
              "id": "HP_0030084"
            },
            "phenotypeEFO": {
              "id": "HP_0030084"
            }
          },
          {
            "phenotypeHPO": {
              "id": "HP_0002023"
            },
            "phenotypeEFO": null
          }
        ]
      }
    }
  }
}
```
</details>

The Clinical Signs and Symptoms widget is based on the `diseaseToPhenotype` dataset, in which we find relationships between diseases in EFO and phenotypes in HPO (some of these might as well be mapped to EFO). Therefore **`phenotypeHPO` should never be null.**

However, it seems like this is not coming from the data, as there are no records where `phenotype` is null. There are 51 reported phenotypes and none of them look weird to me. For some reason, the data is not being properly ingested in the API. @mbdebian, could you look at it?

**Comment by d0choa** [comment]
Likely caused by #2778

**Comment by d0choa** [comment]
@chinmehta I just discussed this with @carcruz. Could you try to handle the `null`?

The data is wrong, but it would be good if we don't crash the widget

**Comment by chinmehta** [comment]
preview: https://deploy-preview-177--ot-platform.netlify.app/disease/EFO_0001064

**Comment by mbdebian** [comment]
I can see this has been fixed for 23.06 release
